ledger/src/blockstore.rs

```diff
@@ -1711,7 +1712,18 @@ impl Blockstore {
                 let slot_transaction_iterator = slot_entries
                     .iter()
                     .cloned()
-                    .flat_map(|entry| entry.transactions);
+                    .flat_map(|entry| entry.transactions)
+                    .map(|transaction| {
+                        if let Err(err) = transaction.sanitize() {
+                            warn!(
+                                "Blockstore::get_confirmed_block sanitize error: {:?}, \
```

Comment:
How about "... sanitize failed:"?  Using the word "error" in warnings seems to confuse people :slightly_smiling_face:

Reply:
Good call! Done.